norse/torch/module/replicate_through_time.py

```diff
@@ -0,0 +1,14 @@
+import torch
+
+class ReplicateThroughTime(torch.nn.Module):
+    def __init__(self, module, T):
+        self.module_list = torch.nn.ModuleList([module for _ in range(T)])
+        self.T = T
+
+    def forward(self, x):
+        outputs = []
+        for ts in range(self.T):
+            y = self.module_list[ts](x[ts])
+            outputs.append(y)
+
+        return torch.stack(outputs)
```

Comment:
This will break if the module returns tuples (output, state).

Reply:
Yes, but this isn't meant for tuples.

Reply:
Rather this is meant for modules which are applied "point wise" in time but with one module per timepoint